src/components/BaremetalInventory.tsx

```diff
@@ -1,37 +1,66 @@
-import React, { FC, Fragment } from 'react';
-
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
 import {
   PageSectionVariants,
   Toolbar,
   ToolbarGroup,
   ToolbarItem,
   Button
 } from '@patternfly/react-core';
+
+import { fetchHostsAsync } from '../actions/hosts';
+import { getHostTableRows, getHostsLoading } from '../selectors/hosts';
+import { RootState } from '../store/rootReducer';
 import PageSection from './ui/PageSection';
 import HostsTable from './HostsTable';
 
-const BaremetalInventory: FC = (): JSX.Element => (
-  <Fragment>
-    <PageSection variant={PageSectionVariants.darker}>
-      Summary stats
-    </PageSection>
-    <PageSection variant={PageSectionVariants.light}>
-      <Toolbar>
-        <ToolbarGroup>
-          <ToolbarItem>
-            <Button variant="primary">Add Hosts</Button>
-          </ToolbarItem>
-        </ToolbarGroup>
-      </Toolbar>
-    </PageSection>
-    <PageSection
-      variant={PageSectionVariants.light}
-      isMain
-      style={{ padding: 0 }}
-    >
-      <HostsTable />
-    </PageSection>
-  </Fragment>
-);
+interface Props {
+  hostRows: string[][];
```

Comment:
Let's use a more concrete type here instead of `string[][]`

Reply:
done